zebra-rpc/src/methods/get_block_template_rpcs.rs

```diff
@@ -347,33 +349,38 @@ where
 
             // The loop returns the server long poll ID,
             // which should be different to the client long poll ID.
-            let (server_long_poll_id, chain_tip_and_local_time, mempool_txs) = loop {
+            let (server_long_poll_id, chain_tip_and_local_time, mempool_txs, submit_old) = loop {
```

Comment:
Thanks, I already added optional TODOs for these (and for sync status, which we might also want to check using a watch channel.)

But this isn't a priority for miners, so I think it is optional and out of scope for now. If we want to do it, let's open a separate ticket, so we can schedule it separately.

I also think this might take significant changes to the rest of Zebra to get these channels to RPC initialisation in the start command. So we might want to delay it until after the audit. Or until a miner really wants it.